phpunit.xml.dist

```diff
@@ -14,7 +14,7 @@
     <testsuites>
         <testsuite name="SemanicResultFormats">
             <directory>tests/phpunit</directory>
-            <exclude>tests/phpunit/ResourcesTest.php</exclude>
```

Comment:
I'm guessing https://github.com/SemanticMediaWiki/SemanticResultFormats/commit/1eb0b5522db0972815fdb9ec88d1b2a5a8bef682 this to be the culprit.